public/assets/scss/_index.scss

```diff
@@ -490,6 +507,10 @@ h5 .btn-link {
     .faq {
         &-card-heading {
             font-size: 1.6rem;
+
+            .first {
```

Comment:
Can you use CSS first-child selector instead of adding a `first` class? This will make the code  maintainable. https://www.w3schools.com/cssref/sel_firstchild.asp

Reply:
this doesn't work for this case

Reply:
I am not sure you went the link above to totally understand how first-child work? It worked when I tried it. See the code below
```css
  .faq {
    .card:first-child span {
      padding-top: 1rem;
    }
    &-card-heading {
      font-size: 1.6rem;
    }
    &-icon {
      font-size: 2.5rem;
    }
  }
```